destinations/data/betterstack.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: betterstack
+  displayName: Better Stack
+  category: managed
+spec:
+  image: betterstack.svg
+  signals:
+    traces:
+      supported: false
+    metrics:
+      supported: true
+    logs:
+      supported: true
+  fields:
+    - name: BETTERSTACK_METRICS_ENDPOINT
+      displayName: OTLP HTTP Metrics Endpoint
+      componentType: input
+      initialValue: 'https://in-otel.logs.betterstack.com/metrics'
+      componentProps:
+        type: text
+        required: true
```

Comment:
If I use this destination for logs only, this setting will make it so that one must fill this value even though it is not relevant and not going to be used